### DISCUSSION TITLE
How to set primary key index sort order in MySQL?

### DISCUSSION THREAD

**Discussion by deepinder10** [prisma/prisma#14797] [Q&A]
Since MYSQL supports descending indexes, I need to sort my primary key in descending order something like this
`PRIMARY KEY (`id` DESC),`
How can I do it with the schema?

**Comment by janpio** [comment]
That sounds pretty close to https://www.prisma.io/docs/concepts/components/prisma-schema/indexes#configuring-the-index-sort-order-with-sort, although that does not support MySQL primary keys (`@id` and `@@id`). Did we miss something there? Can you link to some MySQL docs on this functionality so we can see if this is the same as on SQL Server?

**Reply by deepinder10** [reply]
Thanks for replying. MySQL started supporting descending indexes from v8.
https://dev.mysql.com/doc/refman/8.0/en/descending-indexes.html
We can also set the primary key to be desc, but I have to run a ALTER command to do that.

**Reply by janpio** [reply]
Ok cool. Can you please open a feature request issue for this and link to it from here? Then we can take that into our list.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
